amodeus/upstream/credentials.py

```diff
@@ -106,9 +106,8 @@ async def login(cls, login: str, password: str) -> ModeusCredentials:
         session = AsyncClient(http2=True, base_url="https://utmn.modeus.org/", timeout=15)
         # Getting app config
         r = await session.get("/schedule-calendar/assets/app.config.json")
-        data = r.json()["legacy"]["appConfig"]["httpAuth"]
-        auth_url = data["authUrl"]
-        client_id = data["clientId"]
+        client_id = r.json()["wso"]["clientId"]
+        auth_url = r.json()["wso"]["loginUrl"]
```

Comment:
Please don't decode the response twice.

```suggestion
        data = r.json()["wso"]
        auth_url = data["loginUrl"]
        client_id = data["clientId"]
```